VerificationCounterFunction/RequestVerificationLink.js

```diff
@@ -1,22 +1,12 @@
-const requestVerificationLink = () => {
-  // Disable the button to prevent multiple requests
-  document.getElementById("requestLinkButton").disabled = true;
-
-  // Set the initial count
-  let count = 120;
-
-  // Update the button text and countdown every second
-  const countdown = setInterval(() => {
-    count--;
-    document.getElementById(
-      "requestLinkButton"
-    ).innerText = `Request Link (${count}s)`;
-
-    // If the countdown reaches 0, re-enable the button
-    if (count === 0) {
-      clearInterval(countdown);
-      document.getElementById("requestLinkButton").disabled = false;
-      document.getElementById("requestLinkButton").innerText = "Request Link";
+const startCounter = () => {
+  let counter = 120;
+  const intervalID = setInterval(() => {
+    counter--;
+    if (counter == 0) {
+      clearInterval(intervalID);
+      return true;
     }
   }, 1000);
+  return false;
 };
+startCounter();
```

Comment:
We don't need this

Rather do a named export 

`export {startCounter};`

Reply:
Hi @Zer0SZN, you can turn your function to a named export like so:

```
function startCounter() {
  let counter = 120;
  const intervalID = setInterval(() => {
    counter--;
    if (counter == 0) {
      clearInterval(intervalID);
      return true;
    }
  }, 1000);
  return false;
};

export {startCounter};
```

Reply:
Yea thanks bro